Makefile

```diff
@@ -65,6 +65,7 @@ modules      += extern/AmbiqSuite/$(AS_VERSION)/third_party/cordio
 endif
 
 ifeq ($(USB_PRESENT),1)
+modules      += extern/AmbiqSuite/$(AS_VERSION)/third_party/cordio
```

Comment:
this isn't going to work - for example, Apollo5 Ambiqsuite versions don't have the cordio (they don't have BLE). Instead, copy the needed headers and C to a neuralspot directory (since this is about sending packets around, maybe ns-core or ns-ipc are good homes for CRC)